
The usual label at end of compound statement fixes for gcc-3.4.

  - Rene Rebe <rene@rocklinux.org>

--- ./lib/inet_sr.c.vanilla	2004-07-04 20:31:07.000000000 +0200
+++ ./lib/inet_sr.c	2004-07-04 20:31:15.000000000 +0200
@@ -104,7 +104,6 @@
        isnet = 1; break;
     case 2:
        isnet = 0; break;
-    default:
     }
 
     /* Fill in the other fields. */
--- ./hostname.c.vanilla	2004-07-04 20:31:45.000000000 +0200
+++ ./hostname.c	2004-07-04 20:32:07.000000000 +0200
@@ -97,7 +97,6 @@
 	case EINVAL:
 	    fprintf(stderr, _("%s: name too long\n"), program_name);
 	    break;
-	default:
 	}
 	exit(1);
     };
@@ -116,7 +115,6 @@
 	case EINVAL:
 	    fprintf(stderr, _("%s: name too long\n"), program_name);
 	    break;
-	default:
 	}
 	exit(1);
     };
@@ -173,7 +171,6 @@
 	    *p = '\0';
 	printf("%s\n", hp->h_name);
 	break;
-    default:
     }
 }
 
